cheatsheets/Mobile_Application_Security_Cheat_Sheet.md

```diff
@@ -41,7 +41,7 @@ Developing with third-party libraries and components introduces the possibility
 of security unknowns.
 
 - Ensure app signing.
-- Use only trusted and validated third-party libraries & components.
+- Use only trusted and validated third-party libraries and components.
```

Comment:
Why such a change? Does it improve the readability?

Reply:
This is better english. You do not want to use & in formal language to replace "and".